technical-translation-glossary/bitcoin_terminology.csv

```diff
@@ -0,0 +1,56 @@
+en,es,fr
+bitcoin,bitcoin,bitcoin
+blockchain,blockchain,blockchain
+proof of work,prueba de trabajo,preuve de travail
+proof-of-work,prueba de trabajo,preuve de travail
+offchain,offchain,offchain
+onchain,onchain,onchain
+coinbase,coinbase,coinbase
+cold storage,cold storage,cold storage
+hash,hash,hash
+hashed time lock contract,"hashed time lock contract","hashed time lock contract"

```

Comment:
why hash time lock contracts in quotes when none of the other terms are?

Reply:
was chasing down a whitespace error and forgot to revert this.